.github/workflows/cron-sync-core.yaml

```diff
@@ -30,6 +30,8 @@ jobs:
       # Note we assume a simple version update so no chart version update
       - run: |
           sed -i "s/^appVersion.*$/appVersion: $NEW_CALYPTIA_CORE_VERSION/g" ./charts/core/Chart.yaml
+          NEW_CALYPTIA_CORE_VERSION_="${NEW_CALYPTIA_CORE_VERSION:1}"
```

Comment:
It would be helpful to show the worked example and the fact this relies on the `v` prefix being present - initially I was confused as it just removes the first character in the string:
```
$ export NEW_CALYPTIA_CORE_VERSION=v1.1.1
$ export NEW_CALYPTIA_CORE_VERSION_="${NEW_CALYPTIA_CORE_VERSION:1}"
$ echo $NEW_CALYPTIA_CORE_VERSION_
1.1.1
$ sed "s/^version.*$/version: $NEW_CALYPTIA_CORE_VERSION_/g" ./charts/core/Chart.yaml
apiVersion: v2
name: core
description: A helm chart to deploy Calyptia Inc. core on Kubernetes

type: application
version: 1.1.1
appVersion: v1.1.2
```

I think we probably want to do a regex match here to catch out someone creating an invalid release or breaking our assumptions: https://github.com/calyptia/core-images/blob/23b59978f6a4799b56ef15b4198442129bea6b14/vm/scripts/update-core-version.sh#L23-L29

```bash
if [[ "$NEW_CALYPTIA_CORE_VERSION" =~ ^v?([0-9]+\.[0-9]+\.[0-9])+$ ]] ; then
    NEW_CALYPTIA_CORE_VERSION=${BASH_REMATCH[1]}
    echo "Valid version string: $NEW_CALYPTIA_CORE_VERSION"
else
    echo "ERROR: Invalid semver string: $NEW_CALYPTIA_CORE_VERSION"
    exit 1
fi
```
We should probably get the semver version (and fail if the regex is not matched) then add the `v` prefix for `appVersion` and use directly for `chartVersion`. We only need one variable then.

Reply:
sure, will take a look

Reply:
Added the helper script you pointed and made a minor change in the regex to find valid semver versions. I moved the last + character to inside. The regex we had in there was wrong.

Below are few tests

```
root@c3132faab33a:/# export NEW_CALYPTIA_CORE_VERSION=10.3.1
root@c3132faab33a:/# ./semverCheck.sh
Valid version string: 10.3.1
root@c3132faab33a:/# export NEW_CALYPTIA_CORE_VERSION=10.3.10
root@c3132faab33a:/# ./semverCheck.sh
ERROR: Invalid semver string: 10.3.10
root@c3132faab33a:/# export NEW_CALYPTIA_CORE_VERSION=10.3.10.2.6
root@c3132faab33a:/# ./semverCheck.sh
Valid version string: 0.2.6
```

after adding the regex fix

```
root@c3132faab33a:/# ./semverCheck.sh
Valid version string: 10.3.1
root@c3132faab33a:/# export NEW_CALYPTIA_CORE_VERSION=10.3.10
root@c3132faab33a:/# ./semverCheck.sh
Valid version string: 10.3.10
root@c3132faab33a:/# export NEW_CALYPTIA_CORE_VERSION=10.3.10.2.6
root@c3132faab33a:/# ./semverCheck.sh
ERROR: Invalid semver string: 10.3.10.2.6
```